packages/typespec-azure-core/README.md

```diff
@@ -63,6 +63,7 @@ Available ruleSets:
 | `@azure-tools/typespec-azure-core/response-schema-problem`                                                                                                   | Warn about operations having multiple non-error response schemas.                                                                                    |
 | `@azure-tools/typespec-azure-core/rpc-operation-request-body`                                                                                                | Warning for RPC body problems.                                                                                                                       |
 | `@azure-tools/typespec-azure-core/spread-discriminated-model`                                                                                                | Check a model with a discriminator has not been used in composition.                                                                                 |
+| `@azure-tools/typespec-azure-core/use-standard-integer`                                                                                                      | Use only types that map to int32 or int64.                                                                                                           |
```

Comment:
This should name the specific types `safeInt` and `int32` and `int64`

Reply:
actually should we recommend `int64`? wouldn't we always want people to use `safeint`

Reply:
The new message just recommends using bit-specific sizes.